octobot_trading/exchanges/exchange_manager.py

```diff
@@ -249,28 +251,27 @@ def _load_config_symbols_and_time_frames(self):
         client = self.exchange.client
         if client:
             self.client_symbols = client.symbols
-            self.client_time_frames[CONFIG_WILDCARD] = client.timeframes if hasattr(client, "timeframes") else {}
+            self.client_time_frames = client.timeframes if hasattr(client, "timeframes") else []
```

Comment:
What about a backtesting file with different time_frame foreach symbol ?

Reply:
I was thinking about it and I think it's not currently possible. If data are from multiple exchanges which can make different timeframes, they will be stored in different files and use different exchange managers. Do you see a case where we can have different available timeframes in the same file ?

Reply:
Not in the same file but in the same backtesting. We wanted to implement multiple data files backtesting. We can imagine, in this case, that we'll have different time frame from the different data files.

Reply:
Right, in this case they either come from the same exchange or different ones. If they come from the same exchange, I think we can consider the same available timeframe for each symbol. If they are from different exchange, they will create different exchange managers that can handle different timeframes on their sides.
If you think we can have multiple datafiles on the same exchange and with different timesframes per symbol, then we should re-implement the dict of timeframes per symbol. I think this case can happen (for ex using a mix of different data files with different sources) but is highly improbable since in backtesting it doesn't make much sense to compare multiple symbol results based on different timeframes (and therefore different evaluation styles etc)

Reply:
Yes it's probably the reason why I implemented the dict of timeframes per symbol. I think we can go for a solution that manage only one backtesting file per exchange for now.